tensorflow_model_optimization/g3doc/guide/clustering/clustering_example.ipynb

```diff
@@ -0,0 +1,515 @@
+{
```

Comment:
<p>"In this section"</p><p> </p><p>"clustering wrapper" is an implementation detail and should not be included.</p><p> </p><p>Imperative style is recommended in TF style guide -&gt; E.g. "Apply clustering to the whole pre-trained model".</p><p><a href="https://github.com/tensorflow/docs/blob/master/tools/templates/notebook.ipynb" target="_blank">https://github.com/tensorflow/docs/blob/master/tools/templates/notebook.ipynb</a></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>